src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrapCheckTest.java

```diff
@@ -35,14 +35,14 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "whitespace" + File.separator + filename);
+                + "whitespace" + File.separator  + "nolinewrap" + File.separator + filename);
     }
 
     @Test
     public void testCaseWithoutLineWrapping() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(NoLineWrapCheck.class);
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputNoLineWrapGood.java"), expected);
+        verify(checkConfig, getPath("InputnolinewrapGood.java"), expected);
```

Comment:
sir i changed the file's location and to make it look difficult i lowercased it. The upper case name would also work but that name was already being used so to prevent any confusion i decided to do so...................